client/src/common/components/CookieConsentBox.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Fragment } from 'react';
+import CookieConsent from 'react-cookie-consent';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import Overlay, { OverlayStyleType } from 'common/components/Overlay';
+
+const CookieConsentBox = ({ isAuthPage, onAccept }) => (
+  <Fragment>
+    {isAuthPage && <Overlay type={OverlayStyleType.MEDIUM} />}
+    <CookieConsent
+      buttonClasses="primary-button"
+      buttonText="Accept"
+      containerClasses="cookie-consent"
+      contentClasses="cookie-consent__content"
+      cookieName="eoc_cookie-consent"
+      disableStyles
+      expires={365}
+      location="bottom"
+      onAccept={onAccept}
+    >
+      We use cookies to ensure you get the best experience. By using this
+      website, you consent to use of these cookies.
+      {isAuthPage && (
+        <Fragment>
+          <br />
+          {'For more information, please see our '}
```

Comment:
check is those braces are needed?

Reply:
I don't know if they are needed but there is no space after word 'our' without braces, and I couldn't find another solution.... this is problem that occures with more complicated text in jsx files. See correction in  privacy policy index.jsx file and my commment